### PR TITLE
Add langterms for 20.20.1

### PR DESCRIPTION
**Pull request by AlexBedley**
Since we were manually editing the files (or using something other than `npm run build`) before this the diff is pretty atrocious. I highly recommend viewing with whitespace ignored.

This PR backports the QE dismiss langterms from https://github.com/BrightspaceHypermediaComponents/activities/pull/570

### CODE REVIEW

File: components/d2l-quick-eval/build/lang/de.js [hunk 1]
```diff
@@ -1,77 +1,78 @@
...
-	'toggleIndicatorLabelActions': 'Aktionen für {target} ausführen',
-	'toggleIndicatorLabelInfo': 'Informationen zu {target} anzeigen',
-	'tryAgain': 'Erneut versuchen',
-	'viewBy': 'Anzeigen nach:',
-	'yes': 'Ja'
-};
+			'activities': 'Aktivitäten',
+			'activityDismissed': 'Aktivität verworfen',
+			'activityName': 'Name der Aktivität',
+			'activityRestored': '{count, plural, =1 {1 Aktivität} other {{Anzahl} Aktivitäten}} wurden wiederhergestellt',
```
Comment (AlexBedley) [inline components/d2l-quick-eval/build/lang/de.js]: @maboelsoudd2l is `count` supposed to be translated here?

File: components/d2l-quick-eval/build/lang/fr.js [hunk 2]
```diff
@@ -1,77 +1,78 @@
...
+			'completed': 'Terminé(e)',
+			'confirmation': 'Confirmation',
+			'courseName': 'Cours',
+			'discussion': 'Discussion',
+			'dismissActivity': 'Rejeter l’activité',
+			'dismissUntil': 'Rejetée jusqu’au...',
+			'dismissedActivities': 'Activités rejetées',
+			'dismissedActivitiesList': 'Liste d’activités rejetées',
+			'dismissedOn': 'Rejetée le {date}',
+			'dismissingAnActivityHides': 'Le fait de rejeter une activité la dissimule d’Évaluation rapide, mais n’affecte pas l’activité',
```
Comment (AlexBedley) [inline components/d2l-quick-eval/build/lang/fr.js]: Is Quick Eval supposed to be translated? I thought product names aren't translatable

File: components/d2l-quick-eval/build/lang/de.js (hunk 1 above)
Comment (maboelsoudd2l) [reply components/d2l-quick-eval/build/lang/de.js]: the correct translation is
{count, plural, =1 {1 Aktivität} other {{count} Aktivitäten}} wurden wiederhergestellt

File: components/d2l-quick-eval/build/lang/fr.js (hunk 2 above)
Comment (maboelsoudd2l) [reply components/d2l-quick-eval/build/lang/fr.js]: good catch, but it seems like this is in an issue in multiple languages (like Arabic and Spanish)

File: components/d2l-quick-eval/build/lang/de.js [hunk 3]
```diff
@@ -1,77 +1,78 @@
...
+			'completed': 'Abgeschlossen',
+			'confirmation': 'Bestätigung',
+			'courseName': 'Kurs',
+			'discussion': 'Diskussion',
+			'dismissActivity': 'Aktivität verwerfen',
+			'dismissUntil': 'Verwerfen bis...',
+			'dismissedActivities': 'Verworfene Aktivitäten',
+			'dismissedActivitiesList': 'Liste verworfener Aktivitäten',
+			'dismissedOn': 'Verworfen am {date}',
+			'dismissingAnActivityHides': 'Wenn Sie eine Aktivität verwerfen, wird sie in Quick Eval ausgeblendet, ohne dass sich dies auf die Aktivität auswirkt',
```
Comment (AlexBedley) [inline components/d2l-quick-eval/build/lang/de.js]: Ah so it turns out Quick Eval should be translated

File: components/d2l-quick-eval/build/lang/ko.js [hunk 4]
```diff
@@ -1,77 +1,78 @@
...
-	'toggleIndicatorLabelActions': '{target}에 작업 수행',
-	'toggleIndicatorLabelInfo': '{target}의 정보 보기',
-	'tryAgain': '다시 시도',
-	'viewBy': '보기 기준:',
-	'yes': '예'
-};
+			'activities': '활동',
+			'activityDismissed': '활동이 해제됨',
+			'activityName': '활동 이름',
+			'activityRestored': '{Count,plural,=1{1개 활동} other {{count}개 활동}}이(가) 복원되었습니다',
```
Comment (AlexBedley) [inline components/d2l-quick-eval/build/lang/ko.js]: hmm the spacing is all messed up here... I'll test this one specifically

File: components/d2l-quick-eval/build/lang/pt.js [hunk 5]
```diff
@@ -1,77 +1,78 @@
...
-	'toggleIndicatorLabelActions': 'Executar ações em {target}',
-	'toggleIndicatorLabelInfo': 'Exibir informações em {target}',
-	'tryAgain': 'Tentar novamente',
-	'viewBy': 'Exibir por:',
-	'yes': 'Sim'
-};
+			'activities': 'Atividades',
+			'activityDismissed': 'Atividade Dispensada',
+			'activityName': 'Nome da Atividade',
+			'activityRestored': 'Atividades restauradas: {count, plural, =1 {1} other {{count}}}',
```
Comment (AlexBedley) [inline components/d2l-quick-eval/build/lang/pt.js]: this one just looks broken.... but I have no idea how to speak Portuguese

File: components/d2l-quick-eval/build/lang/tr.js [hunk 6]
```diff
@@ -1,77 +1,78 @@
...
-	'toggleIndicatorLabelActions': '{target} Üzerinde Eylemler Gerçekleştir',
-	'toggleIndicatorLabelInfo': '{target} ile ilgili bilgileri görüntüleyin',
-	'tryAgain': 'Tekrar Dene',
-	'viewBy': 'Görüntüleme ölçütü:',
-	'yes': 'Evet'
-};
+			'activities': 'Etkinlikler',
+			'activityDismissed': 'Etkinlik Çıkarıldı',
+			'activityName': 'Etkinlik Adı',
+			'activityRestored': '{count, plural, =1 {1 activity} other {{count} activities}} yeniden yüklendi',
```
Comment (AlexBedley) [inline components/d2l-quick-eval/build/lang/tr.js]: this doesn't look right either

File: components/d2l-quick-eval/build/lang/tr.js [hunk 7]
```diff
@@ -1,77 +1,78 @@
...
+			'completed': 'Tamamlandı',
+			'confirmation': 'Onay',
+			'courseName': 'Ders',
+			'discussion': 'Tartışma',
+			'dismissActivity': 'Etkinliği Çıkar',
+			'dismissUntil': 'Şu zamana kadar çıkar...',
+			'dismissedActivities': 'Çıkarılan etkinlikler',
+			'dismissedActivitiesList': 'Çıkarılan Etkinliklerin Listesi',
+			'dismissedOn': '{date} tarihinde çıkarıldı',
+			'dismissingAnActivityHides': 'Bir etkinliğin çıkarılması, etkinliği Quick Eval\'den gizler ancak etkinliği etkilemez',
```
Comment (AlexBedley) [inline components/d2l-quick-eval/build/lang/tr.js]: this should be translated

File: components/d2l-quick-eval/build/lang/zh-tw.js [hunk 8]
```diff
@@ -1,77 +1,78 @@
...
-	'toggleIndicatorLabelActions': '對 {target} 執行動作',
-	'toggleIndicatorLabelInfo': '檢視 {target} 相關資訊',
-	'tryAgain': '再試一次',
-	'viewBy': '檢視依據：',
-	'yes': '是'
-};
+			'activities': '活動',
+			'activityDismissed': '活動已關閉',
+			'activityName': '活動名稱',
+			'activityRestored': '已還原 {count, plural, =1 {1 activity} other {{count} activities}}',
```
Comment (AlexBedley) [inline components/d2l-quick-eval/build/lang/zh-tw.js]: missed the translation

File: components/d2l-quick-eval/build/lang/zh-tw.js [hunk 9]
```diff
@@ -1,77 +1,78 @@
...
+			'evaluate': '評估{displayName}',
+			'evaluateAll': '全部評估',
+			'evaluated': '已評估',
+			'failedToDismissActivity': '不能關閉活動。請再試一次。',
+			'failedToFilter': '無法套用篩選器。請在幾分鐘後再試一次。',
+			'failedToLoadActivities': '無法載入活動。請在幾分鐘後再試一次。',
+			'failedToLoadData': '無法載入提交項目。請在幾分鐘後再試一次。',
+			'failedToLoadMore': '無法載入更多提交項目。請在幾分鐘後再試一次。',
+			'failedToRestoreActivity': '活動無法還原。請再試一次',
+			'failedToRestoreSomeActivities': '共有 {y} 個活動，{x} 個無法還原。請再試一次',
```
Comment (AlexBedley) [inline components/d2l-quick-eval/build/lang/zh-tw.js]: having the denominator come first in some of these languages makes me nervous but what do I know

File: components/d2l-quick-eval/build/lang/pt.js (hunk 5 above)
Comment (maboelsoudd2l) [reply components/d2l-quick-eval/build/lang/pt.js]: lol I think the translator was trying to say 
"Activities restored: 38"
and completely ignored the whole plural thing

File: components/d2l-quick-eval/build/lang/tr.js (hunk 6 above)
Comment (maboelsoudd2l) [reply components/d2l-quick-eval/build/lang/tr.js]: 🤷‍♂

File: components/d2l-quick-eval/build/lang/pt.js (hunk 5 above)
Comment (AlexBedley) [reply components/d2l-quick-eval/build/lang/pt.js]: hmm then it might be fine. I'll give them the benefit of the doubt

**Comment by AlexBedley** [comment]
> we need to keep track of what still needs to be translated

I put it in the slack thread, I'll formalize it into Rally soon

**Pull request merged**